package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "supertokens-auth-react",
-    "version": "0.17.6",
+    "version": "0.17.7",
```

Comment:
You also need to edit package-lock. Did you install pre commit hook?

Reply:
Done